pages.ko/common/git-annotate.md

```diff
@@ -0,0 +1,12 @@
+# git annotate
+
+> 각각의 파일란에 커밋한 해쉬와 마지막 작성자를 보여 줍니다. 깃 어노테이트 보다 많이 사용되는 깃 블레임을 살펴 보세요. 깃 어노테이트는 다른 버전 관리 시스템에 친숙한 분들께 제공됩니다.
+> 더 많은 정보: <https://git-scm.com/docs/git-annotate>.
+
+- 각각의 라인에 작성자의 이름과 커밋 해쉬를 앞쪽에 더하여 파일 출력:
+
+`git annotate {{path/to/file}}`
```

Comment:
Please translate the placeholders, like `path/to/file`, for get this easier for non-English speakers.

Reply:
@ellijs Refer #8820 if necessary